apps/api/lib/viewCollection.js

```diff
@@ -32,15 +32,17 @@ export function construct({ client }) {
 
     // TODO: Construct generated from NodeShape
     return CONSTRUCT`
-      ?resource ${schema.identifier} ?id ; 
+      ?resource a ?type ;
+                ${schema.identifier} ?id ; 
```

Comment:
Slightly unrelated, without the RDF type, the UI cannot discover that a view is deletable